### PR TITLE
ZEN-31092 [RCE] Remote Code Execution with weak SUDO configuration (D…

### PR DESCRIPTION
**Pull request by ISPHOST**
…evice Administration )

### CODE REVIEW

**Comment by ISPHOST** [comment]
Removes script var_chown because this script not uses.

**Comment by ISPHOST** [comment]
PR for product assembly https://github.com/zenoss/product-assembly/pull/858

File: etc/sudoers.d/zenoss_nmap [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Allows Zenoss to use nmap for pinging
-%zenoss ALL=(ALL) NOPASSWD: /usr/bin/nmap 
+%zenoss ALL=(ALL) NOPASSWD: /usr/bin/nmap !(*—script*) !(*-sC*)
```
Comment (jplouis) [inline etc/sudoers.d/zenoss_nmap]: Can you explain what this does?

Comment (ISPHOST) [reply etc/sudoers.d/zenoss_nmap]: Deny use -script in args line, because users can runs (/bin/bash) as nmap script file as root user

**Comment by zedieck** [comment]
Since this change was to the CZ side, I moved the project to ZEN from ZING and changed the names to reflect the move.

**Pull request merged**